specification/search/resource-manager/Microsoft.Search/preview/2019-10-01-preview/search.json

```diff
@@ -1106,6 +1423,190 @@
+          "items": {
+            "$ref": "#/definitions/PrivateEndpointConnection"
+          },
+          "description": "The list of Private Endpoint connections."
+        }
+      },
+      "description": "Response containing a list of Private Endpoint connections.",
+      "x-ms-external": true
+    },
+    "SharedPrivateLinkResource": {
+      "properties": {
+        "name": {
+          "type": "string",
+          "description": "The name of the shared private link resource.",
+          "externalDocs": {
+            "url": "https://aka.ms/search-naming-rules"
+          }
+        },
+        "properties": {
+          "$ref": "#/definitions/SharedPrivateLinkResourceProperties",
+          "description": "Describes the properties of a Shared Private Link Resource managed by the Azure Cognitive Search service."
+        }
+      },
+      "description": "Describes a Shared Private Link Resource managed by the Azure Cognitive Search service."
+    },
+    "SharedPrivateLinkResourceProperties": {
+      "properties": {
+        "privateLinkResourceId": {
+          "type": "string",
+          "description": "The resource id of the resource the shared private link resource is for."
+        },
+        "groupId": {
+          "type": "string",
+          "description": "The group id from the provider of resource the shared private link resource is for."
+        },
+        "requestMessage": {
+          "type": "string",
+          "description": "The request message for requesting approval of the shared private link resource."
+        },
+        "status": {
+          "type": "string",
+          "description": "Status of the shared private link resource. Can be Pending, Approved, Rejected, Disconnected, or Timeout.",
+          "enum": [
+            "Pending",
+            "Approved",
+            "Rejected",
+            "Disconnected",
+            "Timeout"
+          ],
+          "x-ms-enum": {
+            "name": "SharedPrivateLinkResourceStatus",
+            "modelAsString": false
+          }
+        }
+      },
+      "description": "Describes the properties of an existing Shared Private Link Resource managed by the Azure Cognitive Search service."
+    },
+    "SharedPrivateLinkResourceListResult": {
+      "properties": {
+        "value": {
+          "readOnly": true,
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/SharedPrivateLinkResource"
+          },
+          "description": "The list of Shared Private Link Resources."
+        }
+      },
+      "description": "Response containing a list of Shared Private Link Resources.",
+      "x-ms-external": true
+    },
+    "PrivateLinkResourcesResult": {
+      "properties": {
+        "value": {
+          "readOnly": true,
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/PrivateLinkResource"
+          },
+          "description": "The list of supported Private Link Resources."
+        }
```

Comment:
>} [](start = 8, length = 1)

add
    "nextLink": {
          "type": "string",
          "description": "The URL to get the next set of operations, if there are any."
        }

Reply:
This particular one I left as is as we chose not to add it in the backend. I presume this is because this return item isn't configurable by the user, and will only ever be a fixed list that we send, and we don't expect it to get so long that paging would be necessary/useful.